lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +212,27 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        conllu_sentence = []
+        for index, sentence in enumerate(split_by_sentence(text)):
+            for text in sentence.split():
+                conllu_list = [ConlluToken(text)]
+                conllu_sentence.append(ConlluSentence(index, sentence, conllu_list))
```

Comment:
Why do you create many sentences with a single token? Tokens from the same original sentence must end up in the same `ConlluSentence` instance